cleaned_data/remediation_files/open_refine_template.md

```diff
@@ -16,23 +16,21 @@
 <identifier type="local">{{cells["identifier"].value}}</identifier>
 <identifier type="pid">{{cells["PID"].value}}</identifier>
 {{if(isBlank(cells['title'].value), '', '<titleInfo><title>' + cells["title"].value + '</title></titleInfo>')}} 
-<titleInfo supplied="yes"><title>{{cells['title_supplied'].value}}</title></titleInfo>
-{{if(isBlank(cells['dateIssued'].value), '', '<originInfo><dateIssued>' + cells['dateIssued'].value + '</dateIssued><dateIssued encoding="edtf" keyDate="yes">' + cells['dateIssued'].value + '</dateIssued><publisher>' + cells['publisher'].value + '</publisher><place><placeTerm valueURI="http://id.loc.gov/authorities/names/n79109786">Knoxville (Tenn.)</placeTerm>
+{{if(isBlank(cells['dateIssued'].value), '', '<originInfo><dateIssued>' + cells['dateIssued'].value + '</dateIssued><dateIssued encoding="edtf" keyDate="yes">' + cells['dateIssued'].value + '</dateIssued><publisher>' + '</publisher><place><placeTerm valueURI="http://id.loc.gov/authorities/names/n79109786">Knoxville (Tenn.)</placeTerm>
 </place></originInfo>')}}
 <abstract>{{cells["abstract"].value}}</abstract>
-<physicalDescription><form authority="aat" valueURI="{{cells['form_URI'].value}}">{{cells['form'].value}}</form><form authority="aat" valueURI="{{cells['form2_URI'].value}}">{{cells['form2'].value}}</form><extent>{{cells["extent"].value}}</extent></physicalDescription>
-{{if(isBlank(cells['note'].value), '', '<note>' + cells['note'].value + '</note>')}} 
+<name{{if(isBlank(cells["artist_URI"].value), '', ' authority="naf" valueURI="' + cells["artist_URI"].value + '"')}}><namePart>{{cells["artist"].value}}</namePart><role><roleTerm authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/art">Artist</roleTerm></role></name>
+{{if(isBlank(cells["interviewee_1"].value), '', '<name>' + '<namePart>' + cells["interviewee_1"].value + '</namePart>' + '<role>' + '<roleTerm authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/ive">Interviewee</roleTerm>' + '</role>' + '</name>')}}
```

Comment:
@logantillman - I wanted to reiterate here that we should have interviewee values in subject headings. No roleTerm needs to be included for them. The path should be subject/name/namePart.